terraform/gcp/modules/fulcio/service_accounts.tf

```diff
@@ -34,3 +34,10 @@ resource "google_project_iam_member" "fulcio_member" {
   member     = "serviceAccount:${google_service_account.fulcio-sa.email}"
   depends_on = [google_service_account.fulcio-sa]
 }
+
+resource "google_project_iam_member" "fulcio_kms_signer_verifier_member" {
+  project    = var.project_id
+  role       = "roles/cloudkms.signerVerifier"
+  member     = "serviceAccount:${google_service_account.fulcio-sa.email}"
+  depends_on = [google_service_account.fulcio-sa]
+}
```

Comment:
For rekor we also needed `roles/cloudkms.viewer` to get the key and the key version -- will fulcio need that as well? [this is the terraform.](https://github.com/sigstore/scaffolding/blob/main/terraform/gcp/modules/rekor/service_accounts.tf#L38-L43).

Reply:
I saw this and was wondering why viewer is needed. Do you recall why it was needed for Rekor? signerVerifier should be sufficient for Fulcio I believe, since it'll only need to grab the public key for verifying signatures, and use the private key for signing.

Reply:
I think `GetCryptoKey` is always run when getting the KMS client from sigstore/sigstore:

https://github.com/sigstore/sigstore/blob/fa3059196b4a215a0b8f9634329dc879f1a2953c/pkg/signature/kms/gcp/client.go#L177

Reply:
Ah that's unexpected. Added viewer.